Add release notes for Deploy 13.1.1 and 14.0.2 (#6261)
umbraco/UmbracoDocs: 14/umbraco-deploy/getting-started/deploy-settings.md

* `SourceDeployBatchSize` - applies a batch setting for the transfer of multiple selected items to an upstream environment (such as a media folder with many images).
* `PackageBatchSize` - applies a batch setting to the processing of a Deploy "package", which contains all the items selected for a Deploy operation, plus all the determined dependencies and relations.

> ### MaxRequestLength

Reviewer: Looks like you haven't added this to the 13 version of this page.  Assuming it's still valid for that version, it should be added there too.

Author: This setting is only introduced in v14 after refactoring the endpoints, so would require backporting first. Might be something to consider, although it's not been something users have requested or caused any issues in the past.

> When Deploy transfers files between environments that can't connect to each other (like a local environment that isn't exposed publicly), it falls back from fetching/downloading the files to uploading them. This is done in fixed-sized chunks, so multiple files can be combined in a single request or a large file into multiple requests. This setting can specify the size of these chuncks **(in bytes)**.

Reviewer: Typo above "chuncks" --> "chunks"

> When Deploy transfers files between environments that can't connect to each other (like a local environment that isn't exposed publicly), it falls back from fetching/downloading the files to uploading them. This is done in fixed-sized chunks, so multiple files can be combined in a single request or a large file into multiple requests. This setting can specify the size of these chuncks **(in bytes)**.

Reviewer 2: ```suggestion
When Deploy transfers files between unconnected environments (like a non-public local environment), it falls back from fetching/downloading the files to uploading them. This is done in fixed-sized chunks, so multiple files can be combined in a single request or a large file into multiple requests. This setting can specify the size of these chunks **(in bytes)**.
```

> If no value is set, Deploy will create 16MB chunks, unless the [CMS `MaxRequestLength` setting](../../umbraco-cms/reference/configuration/runtimesettings.md) is set to a lower value (note that the CMS setting is configured in kilobytes). Setting this to a value higher than the CMS `MaxRequestLength` (multiplied by 1024) will cause a validation error. Similarly, configuring a value that is higher than the [server or infrastructure allows](../../umbraco-cms/reference/configuration/maximumuploadsizesettings.md) will result in file transfers failing.

Reviewer: IIRC these relative links across "sites" within the documentation won't work, and instead you'll need to create an absolute HTML link to the published page on the docs site when linking to CMS documentation.

> If no value is set, Deploy will create 16MB chunks, unless the [CMS `MaxRequestLength` setting](../../umbraco-cms/reference/configuration/runtimesettings.md) is set to a lower value (note that the CMS setting is configured in kilobytes). Setting this to a value higher than the CMS `MaxRequestLength` (multiplied by 1024) will cause a validation error. Similarly, configuring a value that is higher than the [server or infrastructure allows](../../umbraco-cms/reference/configuration/maximumuploadsizesettings.md) will result in file transfers failing.

Reviewer 2: ```suggestion
If no value is set, Deploy will create 16MB chunks, unless the [CMS `MaxRequestLength` setting](../../umbraco-cms/reference/configuration/runtimesettings.md) is lower (CMS setting is configured in kilobytes). Setting a value higher than the CMS `MaxRequestLength` (multiplied by 1024) will cause a validation error. Similarly, configuring a value higher than the [server or infrastructure limits](../../umbraco-cms/reference/configuration/maximumuploadsizesettings.md) will result in file transfers failures.
```
